packages/contracts/deploy/00_managing-dao/00_managing-dao.ts

```diff
@@ -12,6 +12,10 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const {deploy} = deployments;
   const {deployer} = await getNamedAccounts();
 
+  console.log(
```

Comment:
I would also add a comment when the multisig is installed and when the temp permissions are dropped

Reply:
Amended.